recipes/_master_sge_config.rb

```diff
@@ -28,6 +28,20 @@
   mode '0644'
 end
 
+# Copy qconf utils (Downloaded from http://arc.liv.ac.uk/SGE/downloads/qconf_scripts.tar.gz)
+cookbook_file 'qconf_scripts.tar.gz' do
+  path '/opt/sge/util/qconf_scripts.tar.gz'
```

Comment:
I'd like to define a variable for the `/opt/sge` and for the `/opt/sge/util` paths but it could part of another patch